run-checks

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+mkdir -p bin
+( cd bin &&  wget https://niemeyer.s3.amazonaws.com/spread-amd64.tar.gz -O spread.tar.gz && tar xzvf spread.tar.gz )
+
+export RESULTS_FILE=built_architectures.txt
+
+# build core for all the architectures
+output=0
```

Comment:
Maybe `result` or `res` or `exitcode` instead of `output`? output sounds like some textual output to me.

Reply:
done